pontoon/base/views.py

```diff
@@ -886,6 +886,7 @@ def user_data(request):
             "manager_for_locales": list(
                 user.managed_locales.values_list("code", flat=True)
             ),
+            "pm_for_projects": user.contact_for.all(),
```

Comment:
This returns a ProjectQuerySet. Not sure if there's a better way to get a list of slugs besides list comprehension.

Reply:
Yikes!